src/components/TweetView/Tweet.scss

```diff
@@ -0,0 +1,36 @@
+.block {
+  border-bottom: 1px solid #e1e8ed;
+  border-left: 1px solid #e1e8ed;
+  border-right: 1px solid #e1e8ed;
+  padding: 10px 15px;
```

Comment:
can we use relative units for padding? (especially for vertical one).
That will help us to maintain a vertical rhythm even on screens with zoom enabled.